httpmw/logging/logging.go

```diff
@@ -1,18 +1,56 @@
 package logging
 
 import (
+	"context"
 	"log/slog"
 	"net/http"
 	"time"
 
 	"github.com/gorilla/mux"
 )
 
+type contextKey string
+
+const attrsKey contextKey = "slogAttrs"
```

Comment:
Rather than create a key that lumps all to-be-logged information, I think it is better to create one key for each ID. This will be similar to https://github.com/Azure/aks-middleware/blob/8f407085b1b57638c9da429f248ed61c15e9a4d0/requestid/requestid.go#L19 

Benefits:
* The data put into the context can be used by multiple middlewares (ctxlogger, mdforward), not just by this logging one.

Actions:
* Create a separate middleware similiar to the gRPC requestID middleware

After that, the TODOs will be:
* Create middleware similar to ctxlogger, mdforward in gRPC that leverages the operation-id for HTTP.